ArduCopter/GCS_Mavlink.cpp

```diff
@@ -118,6 +118,29 @@ void GCS_MAVLINK_Copter::send_position_target_global_int()
         0.0f); // yaw_rate
 }
 
+void GCS_MAVLINK_Copter::send_position_target_local_ned()
+{
+    const Vector3f target_pos = copter.wp_nav->get_wp_destination();
```

Comment:
What are these going to return when we're in `STABILIZE` mode?  What would someone reviewing a log with such a section in it think?

Reply:
Added check for whether copter is in Guided mode or not